packages/scss/src/components/inputs/textfield/_textfield.material.scss

```diff
@@ -0,0 +1,121 @@
+%input_material {
+	padding-top: _component("textfield.material.padding-top");
+
+	.textfield-input {
+		background: transparent;
+		border-radius: 0;
+		box-shadow: 0 1px 0 0 _component("textfield.border.color");
+		padding: {
+			left: 0;
+			right: 0;
+		}
+
+		&::placeholder {
+			opacity: 0;
+		}
+
+		&:focus {
+			&::placeholder {
+				opacity: 1;
+			}
+
+			~ .textfield-label {
+				font-size: _theme("sizes.small.font-size");
+				top: 0;
+			}
+		}
+	}
+
+	.textfield-label {
+		color: _component("textfield.label.color");
+		font-size: _component("textfield.font-size");
+		font-weight: 400;
+		left: 0;
+		pointer-events: none;
+		position: absolute;
+		top: 1.3rem;
+		width: 100%;
+	}
+
+	.textfield-suffix {
+		position: absolute;
```

Comment:
We might need to come back to that after redefining textfield-suffix in the textfield.scss (probably need a bottom: auto and we can get rid of position: absolute)

Reply:
Refactored suffix ✔️